File: autopilot/v1/style_sheet/update-example-1/update-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Autopilot.V1.Assistant;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var styleSheet = new Dictionary<string, Object>()
        {
            {"style_sheet", new Dictionary<string, Object>()
                {
                    {"voice", new Dictionary<string, Object>()
                        {
                            {"say_voice", "Polly.Joanna"}
                        }}
                }}
        };

        var styleSheetResource = StyleSheetResource.Update(
            styleSheet: styleSheet,
            pathAssistantSid: "UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(styleSheetResource.AssistantSid);
    }
}

File: autopilot/v1/style_sheet/update-example-1/update-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
// DANGER! This is insecure. See http://twil.io/secure
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$style_sheet = $twilio->autopilot->v1->assistants("UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                     ->styleSheet()
                                     ->update(array(
                                                  "styleSheet" => array(
                                                      "style_sheet" => array(
                                                          "voice" => array(
                                                              "say_voice" => "Polly.Joanna"
                                                          )
                                                      )
                                                  )
                                              )
                                     );

print($style_sheet->assistantSid);